sdk/keyvault/azure-keyvault-keys/README.md

```diff
@@ -249,21 +249,21 @@ credential = DefaultAzureCredential()
 key_client = KeyClient(vault_endpoint=vault_endpoint, credential=credential)
 
 # Create an RSA key
-rsa_key = await key_client.create_rsa_key("rsa-key-name", hsm=False, size=2048)
+rsa_key = await key_client.create_rsa_key("rsa-key-name", size=2048)
 print(rsa_key.name)
-print(rsa_key.key_material.kty)
+print(rsa_key.key_type)
 
 # Create an elliptic curve key
-ec_key = await key_client.create_ec_key("ec-key-name", hsm=False, curve="P-256")
+ec_key = await key_client.create_ec_key("ec-key-name", curve="P-256")
 print(ec_key.name)
-print(ec_key.key_material.kty)
+print(ec_key.key_type)
 ```
 
 ### Asynchronously list keys
 This example lists all the keys in the client's vault:
 
 ```python
-keys = key_client.list_keys()
+keys = key_client.list_properties_of_keys()
```

Comment:
As I mentioned OOB, I might change these to key_properties for consistency, but as with my other comments, minor nits.